app/views/content/train-to-be-a-teacher/_nav-cards.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="container">
+    <div class="category__nav-cards">
+        <a href="/train-to-be-a-teacher/if-you-have-a-degree" class="category__nav-card">
+            <h2>If you have a degree</h2>
+			
+            <p>
+                Find out how to get QTS if you have a degree or you’re studying for one now. There are different options you can choose.
+            </p>
+            <span class="icon"></span>
+        </a>
+
+        <a href="/train-to-be-a-teacher/if-you-have-experience" class="category__nav-card">
+            <h2>If you have classroom experience</h2>
+
+            <p>
+                If you have already spent time working in the classroom, you may be able to follow the assessment only route to show you meet the standards for QTS.
```

Comment:
@valentine-carter I think we need to make it clearer that you need to have significant experience (not necessarily that wording) - this currently (implicitly) implies that this route is open to people with only a little experience

Reply:
this also feels very high up in order - the AO page only gets just over 1% of our pageviews and is unlikely to be appropriate for the majority of our users / doesn't fit with the objective of getting people into ITT?